packages/run/src/index.ts

```diff
@@ -51,7 +51,7 @@ export default function run(opts: RollupRunOptions = {}): Plugin {
       const dir = outputOptions.dir || dirname(outputOptions.file!);
       const entryFileName = Object.keys(bundle).find((fileName) => {
         const chunk = bundle[fileName] as RenderedChunk;
-        return chunk.isEntry && chunk.facadeModuleId === input;
+        return chunk.isEntry;
```

Comment:
There is not much use determining `input` above and then not use it. While we only have a single input, it is not guaranteed that there is not some plugin using `this.emitFile` to add additional entry chunks. Only we do not want to run those chunks.
But we know the module id of the entry point we are interested in. So we can just look into those chunks to figure out which contains the entry module:

```js
return chunk.modules && Object.keys(chunk.modules).includes(input);
```

Also, you should add some test, probably including the commonjs plugin to see if this actually works.